src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -145,6 +145,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <!-- No OpenSSL on Apple platforms or Android -->
       <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.OpenSsl" Condition="'$(StaticOpenSslLinking)' != 'true' and '$(_IsApplePlatform)' != 'true' and '$(_targetOS)' != 'android'" />
       <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.Android" Condition="'$(_targetOS)' == 'android'" />
+      <NetCoreAppNativeLibrary Include="System.IO.Ports.Native" Condition="'$(_targetOS)' == 'android'" />
```

Comment:
`System.IO.Ports` is an independent nuget package that is versioned independenly from the runtime. We should not be pulling in `System.IO.Ports` into native AOT. Instead, we should disable `System.IO.Ports` tests for mobile if they are not disabled already. 

Supporting System.IO.Ports properly would require https://github.com/NuGet/Home/issues/9631 .